docs/mkdocs.yml

```diff
@@ -945,6 +946,80 @@ nav:
           - ./api_reference/instrumentation/index.md
           - ./api_reference/instrumentation/span_handlers.md
           - ./api_reference/instrumentation/span_types.md
+      - Llama Datasets:
+          - ./api_reference/llama_dataset/index.md
+      - Llama Deploy:
+          - ./api_reference/llama_deploy
```

Comment:
Can we add a header for messages queues? For now, it copies the folder name it seems
<img width="247" alt="image" src="https://github.com/user-attachments/assets/58578b3a-2f74-4d8b-b2df-953d985b6495">

Reply:
In order to rely on the `include_dir_to_nav` plugin for "autodiscovery" the docs structure, I changed the folder names on llama_deploy to achieve the same result as using an explicit header in this file. For consistency, I'm using the same strategy in the `mkdocs.yml` file on llama_deploy, so that when you preview the documentation there, you'll know how it'll look once merged here. Not great, but it works with the current stack.